assets/js/modules/idea-hub/components/dashboard/DashboardIdeasWidget/index.stories.js

```diff
@@ -235,6 +240,44 @@ DataUnavailableAll.decorators = [
 	},
 ];
 
+export const Activity = Template.bind( {} );
+Activity.storyName = 'Activity In Progress';
+Activity.decorators = [
+	( Story ) => {
+		const setupRegistry = ( registry ) => {
+			registry
+				.dispatch( MODULES_IDEA_HUB )
+				.setActivity(
+					'ideas/17450692223393508734',
+					IDEA_HUB_ACTIVITY_IS_DELETING
+				);
+			registry
+				.dispatch( MODULES_IDEA_HUB )
+				.setActivity(
+					'ideas/14025103994557865535',
+					IDEA_HUB_ACTIVITY_IS_PINNING
+				);
+			registry
+				.dispatch( MODULES_IDEA_HUB )
+				.setActivity(
+					'ideas/7612031899179595408',
+					IDEA_HUB_ACTIVITY_CREATING_DRAFT
+				);
+		};
+
+		enabledFeatures.clear();
+		enabledFeatures.add( 'ideaHubModule' );
+		mockEndpoints();
+
+		const registry = bootstrapRegistry();
+		return (
+			<WithTestRegistry registry={ registry } callback={ setupRegistry }>
```

Comment:
This should use `WithRegistrySetup` as `WithTestRegistry` is more for Jest now (also perhaps why features may not be enabled as expected).